starter_code/express-spotify-app/app.js

```diff
@@ -0,0 +1,47 @@
+//instantiate express application
+const express         = require('express');
+const app             = express();
+const bodyParser      = require('body-parser');
+const expressLayouts  = require('express-ejs-layouts');
+
+//const expressLayouts = require('express-ejs-layouts');
+
+//SpotifyWebApi
+const SpotifyWebApi = require('spotify-web-api-node');
+const spotifyApi = new SpotifyWebApi({
+  clientId : '44010391e08c4ed88f1db7339ed9601b',
+  clientSecret : '34194b9295f8401f9905ea8719bf5862'
+});
+
+// Retrieve an access token.
+spotifyApi.clientCredentialsGrant()
```

Comment:
This piece of code is asynchronous, so you can have problems if the response is not ready when the app needs it. You can try to wrap it in a function with a callback.